client/components/signup-form/index.jsx

```diff
@@ -324,11 +327,17 @@ class SignupForm extends Component {
 		} );
 	};
 
-	getNoticeMessageWithLogin( notice ) {
-		const link = login( {
+	loginLink() {
```

Comment:
Maybe prefix that with `get` => `getLoginLink()` to clarify that method returns a login link